src/helpers/programHelper.js

```diff
@@ -630,6 +630,20 @@ class ProgramServiceHelper {
 
     const collections = _.get(data, 'config.collections');
     const collectionIds = _.map(collections, 'id');
+    if(_.includes(_.get(data, 'config.contentAdditionMode'), 'new')) {
```

Comment:
@bharatkashyap are you storing `contentAdditionMode` in program config?

Reply:
Yes @pallakartheekreddy

Reply:
@pallakartheekreddy - Have updated this. Please review